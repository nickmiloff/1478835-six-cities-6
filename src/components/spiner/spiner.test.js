import React from 'react';
import {render} from '@testing-library/react';

import Spiner from './spiner';

describe(`Spiner component test`, () => {
  it(`Spiner component' should render correctly`, () => {
    const {container} = render(<Spiner />);

    expect(container).toMatchSnapshot();
  });
});
