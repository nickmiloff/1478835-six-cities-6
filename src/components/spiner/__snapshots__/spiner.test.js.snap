// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spiner component test Spiner component' should render correctly 1`] = `
<div>
  <section
    class="spiner"
  >
    <div
      class="spiner__wrapper"
    >
      <div
        class="spiner__spiner"
      >
        <div />
      </div>
    </div>
  </section>
</div>
`;
